system/Entity.php

```diff
@@ -140,11 +140,12 @@ public function fill(array $data = null)
 	 *
 	 * @param boolean $onlyChanged If true, only return values that have changed since object creation
 	 * @param boolean $cast        If true, properties will be casted.
+	 * @param boolean $deep		   If true, inner entities will be casted as array as well.
```

Comment:
Just fixing alignment.
```suggestion
	 * @param boolean $deep        If true, inner entities will be casted as array as well.
```